src/core/formulations.jl

```diff
@@ -121,21 +121,32 @@ Branch formulation for PhaseShiftingTransformer flow control
 struct PhaseAngleControl <: AbstractBranchFormulation end
 
 ############################### DC Branch Formulations #####################################
-abstract type AbstractDCLineFormulation <: AbstractBranchFormulation end
+abstract type AbstractTwoTerminalDCLineFormulation <: AbstractBranchFormulation end
 """
 Branch type to avoid flow constraints
 """
-struct HVDCP2PUnbounded <: AbstractDCLineFormulation end
+struct HVDCTwoTerminalUnbounded <: AbstractTwoTerminalDCLineFormulation end
 """
 Branch type to represent lossless power flow on DC lines
 """
-struct HVDCP2PLossless <: AbstractDCLineFormulation end
+struct HVDCTwoTerminalLossless <: AbstractTwoTerminalDCLineFormulation end
 """
 Branch type to represent lossy power flow on DC lines
 """
-struct HVDCP2PDispatch <: AbstractDCLineFormulation end
+struct HVDCTwoTerminalDispatch <: AbstractTwoTerminalDCLineFormulation end
 # Not Implemented
-# struct VoltageSourceDC <: AbstractDCLineFormulation end
+# struct VoltageSourceDC <: AbstractTwoTerminalDCLineFormulation end
+
+############################### AC/DC Converter Formulations #####################################
+abstract type AbstractConverterFormulation <: AbstractDeviceFormulation end
+
+"""
+LossLess InterconnectingConverter Model
+"""
+struct LossLessConverter <: AbstractConverterFormulation end
+
+# TODO: Think if this an ok abstraction for future use cases
```

Comment:
Does this need to be resolved in this PR?

Reply:
No, this will be useful for other things later